index.scss

```diff
@@ -198,14 +199,29 @@ $media: null !default;
 
 
 
+/// Checks whether given value is a number with px unit.
+/// @access private
+/// @arg {Any} $value - a value to check
+/// @return {Bool} whether the value is a number with px unit
+/// @example sass
+///   is-number-px(100px) // => true
+///   is-number-px(100em) // => false
+///   is-number-px(100) // => false
+///   is-number-px('text') // => false
+@function is-number-px($value) {
+  @return meta.type-of($value) == 'number' and unit($value) == 'px';
```

Comment:
You're using two functions here, one namespaced and one not.

Ideally, all function calls should be namespaced for clarity.

Reply:
oh that must have slipped my mind after i got distracted looking at `map-sort-values` where none of the `list` functions are namespaced as opposed to everywhere else.

i will fix that here.
should i add another commit that namespaces the function calls in `map-sort-values` ?

Reply:
> should i add another commit that namespaces the function calls in map-sort-values ?

Yes please! 🙇

Reply:
done and done.

i thought you did that on purpose as not to change the code that you attributed to be taken from somewhere else :)

Reply:
I copied it over without realizing the difference in code style.

But I think I'd rather keep the style consistent. There is no benefit in preserving original code unchanged. 🤷 

Thanks again!